docs/index.rst

```diff
@@ -6,50 +6,27 @@ Scripture Burrito Documentation
 
 .. image:: ../logo/burrito_logo.png
 
-This is the official documentation for the evolving Scripture Burrito standard. See the development `milestones here <https://github.com/bible-technology/scripture-burrito/milestones>`_).
+After several years of development and testing, we are pleased to announce the availability of **Scripture Burrito 1.0-rc** specification! We recommend that developers of Scripture and Scripture-related applications test and adopt this specification for interchanging data with other systems. Out of the box Scripture Burrito is designed to support the following types of data:
 
-**This is 1.0.0-beta2** based on `JSON Schema <https://json-schema.org/>`_.
+- :ref:`scripture_text_flavor`
+- :ref:`scripture_print_flavor`
+- :ref:`scripture_audio_flavor`
+- :ref:`scripture_sign_language_flavor`
+- :ref:`scripture_braille_flavor`
+- :ref:`scriptural_text_stories_flavor`
 
-Feedback
---------
+We fully expect that many more flavors will need to be developed. We have provided instructions and examples for Extending Scripture Burrito by testing and implementing new flavors. As new flavors are needed by multiple applications we will consider adding them as official flavors in new schema releases.
 
-Feedback may be provided via the `Technology.Bible forum <https://technology.bible/c/sb>`_ or via the `Scripture Burrito Github issues <https://github.com/bible-technology/scripture-burrito/issues>`_.
+This work has been a multi-year collaboration between several organizations, including American Bible Society, Clear.Bible, Eldarion, BCS/Geneza, SIL, unfoldingWord, United Bible Societies, and the work has been sponsored by illuminations.
 
```

Comment:
@joelthe1 What's the right affiliation for you?  BCS?  Geneza?  Something else?

Reply:
It should be `Bridge Connectivity Solutions`.